app/views/declaration/summary/item_section.scala.html

```diff
@@ -14,7 +14,10 @@
  * limitations under the License.
  *@
 
-@import models.DeclarationType._
+@import views.html.declaration.summary.onward_supply_relief
+@import views.html.declaration.summary.package_information
+@import views.html.declaration.summary.union_and_national_codes
```

Comment:
Why all the new imports?

Reply:
Twirl templates without explicit view imports within IntelliJ Ultimate are flagged as errors.
If we explicitly import them this goes away.
I know that this is meaningless

Reply:
That is a useful tip!